sdks/python/test-suites/portable/common.gradle

```diff
@@ -98,7 +98,7 @@ task portableLocalRunnerJuliaSetWithSetupPy {
   }
 }
 
-task portableLocalRunnerJuliaSetWithRequirementsFile {
+task portableLocalRunnerTestWithRequirementsFile {
```

Comment:
I think you'd need to include this suite into postcommit suite manually.
See: https://github.com/apache/beam/blob/64581e0b5a1ff9c1bb39f3b9269714016f97963f/sdks/python/test-suites/portable/common.gradle#L190
Then, run the test on the PR.

Reply:
Alternatively, we could try to use pytest and define it as integration test, similar to how we define most of our other integration tests, such as one defined in https://github.com/apache/beam/pull/16154. If we do that, I think we may need to restrict it to specific runners.

Reply:
the test-suites works for this case now. Do you want me to mark it as pytest integration test or let it be in the gradle test-suites?

Reply:
I think we should look into making this and setup.py test as pytests since they stand out as snowflakes, but i am fine to do it later.